tests/robustness/traffic/etcd.go

```diff
@@ -65,56 +65,56 @@ var (
 			{Choice: Put, Weight: 40},
 		},
 	}
-	EtcdDelete Traffic = etcdTraffic{
-		keyCount:     10,
-		largePutSize: 32769,
-		leaseTTL:     DefaultLeaseTTL,
+	EtcdDelete Traffic = EtcdTraffic{
+		KeyCount:     10,
+		LargePutSize: 32769,
+		LeaseTTL:     DefaultLeaseTTL,
 		// Please keep the sum of weights equal 100.
-		requests: []random.ChoiceWeight[etcdRequestType]{
+		Requests: []random.ChoiceWeight[EtcdRequestType]{
 			{Choice: Put, Weight: 50},
 			{Choice: Delete, Weight: 50},
 		},
 	}
 )
 
-type etcdTraffic struct {
-	keyCount     int
-	requests     []random.ChoiceWeight[etcdRequestType]
-	leaseTTL     int64
-	largePutSize int
+type EtcdTraffic struct {
+	KeyCount     int
```

Comment:
Making the struct public make sense, however we should consider if it's convenient and safe to be used by users outside this package. It's good to consider that users might not know some internal assumptions about the fields.

Instead of making all fields in struct public we could create a function working as a constructor. This is safer as it allows us to add some logic on construction like validation for fields that would prevent panics. Maybe in the future would be good to check keyCount, LargePutSize and leaseTTL are not negative etcd.

Reply:
Done, I added `NewEtcdTraffic` in adding to exporting `EtcdTraffic`.  I still have the fields public for now.  Let me know if you want me to turn them back to private.